app/actions/remote/entry/common.ts

```diff
@@ -4,7 +4,7 @@
 import {nativeApplicationVersion} from 'expo-application';
 import {RESULTS, checkNotifications} from 'react-native-permissions';
 
-import {fetchAllMyChannelsForAllTeams, fetchMissingDirectChannelsInfo, handleKickFromChannel, type MyChannelsRequest} from '@actions/remote/channel';
```

Comment:
Out of curiosity, why is this being reverted back to fetch channela per team?

Reply:
When this PR is ready for review (out of draft), I will have more details with empirical data why I'm doing it.

Reply:
Fair enough.. I'm just concerned cause this was a huge gain in terms of networking (see https://github.com/mattermost/mattermost-mobile/pull/8420) so it frels going backwards. Not only we reduce the amount of requests but also the amount of duplicate data as every request to get the channels of a team include all dm's and gm's